pwa/src/service-worker/utils.js

```diff
@@ -104,15 +104,23 @@ export async function getClientsInfo(event) {
 
     // Include uncontrolled clients: necessary to know if there are multiple
     // tabs open upon first SW installation
-    const clientsList = await self.clients.matchAll({
-        includeUncontrolled: true,
-    })
+    const filteredClientsList = await self.clients
+        .matchAll({
+            includeUncontrolled: true,
```

Comment:
This is kind of a weird thing: we _do_ sometimes want to include uncontrolled clients that are still in the scope of the service worker, because there are some cases where a client has an active service worker but the client is not controlled, for example during the first installation of the service worker in an app, or after a 'hard reload'. I have no idea why the `includeUncontrolled` option goes on to include all clients on that _domain_ though, even outside the service worker scope 🤷 

Just to be sure, I did an experiment and logged the different results of `matchAll`; here's what it looks like with one uncontrolled tab and another one right after a hard reload -- neither are controlled, but both are in scope and of interest to `getClientsInfo`:
<img width="750" alt="Screen Shot 2022-11-05 at 1 37 41 PM" src="https://user-images.githubusercontent.com/49666798/200120274-42630220-3934-46ba-b48b-739f28d41854.png">